Readability.js

```diff
@@ -867,6 +878,41 @@ Readability.prototype = {
           lastScore = parentOfTopCandidate.readability.contentScore;
           parentOfTopCandidate = parentOfTopCandidate.parentNode;
         }
+
+        // Find a better top candidate node if it contains (at least three) nodes which belong to `topCandidates` array
+        // and whose scores are quite closed with current `topCandidate` node.
+        var topCandidateAncestors = [];
+        topCandidates.some(function(_topCandidate, i) {
+          if (i === 0) {
+            return false;
+          }
+          if (Math.abs((_topCandidate.readability.contentScore -
+                        topCandidate.readability.contentScore) /
+                        topCandidate.readability.contentScore) < 0.25) {
```

Comment:
The subtraction and Math.abs call here is not necessary. The array is sorted, and so the `alternativeCandidate` will always have a lower score. So you can do:

```js
if (alternativeCandidate.readability.contentScore / topCandidate.readability.contentScore >= 0.75) {
   // add ancestors to a set. 
}